specification/communication/data-plane/Identity/stable/2022-10-01/CommunicationIdentity.json

```diff
@@ -23,6 +23,20 @@
           {
             "$ref": "#/parameters/ApiVersionParameter"
           },
+          {
+            "in": "header",
+            "name": "Repeatability-Request-ID",
```

Comment:
It would be nice to add these headers to the example for creating the identity, as taking advantage of them is the best practice.